app/controllers/api/tokens_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::TokensController < Api::ApiController
+  def index
+    @tokens = Token.revoked.order(created_at: :asc)
```

Comment:
Not sure about /api/tokens bringing back just revoked tokens - you'd expect /api/tokens to bring back either all, or the valid tokens, and something like /api/tokens/revoked (or tokens?revoked=1) to be the list of revoked tokens?